src/dxvk/dxvk_graphics.cpp

```diff
@@ -396,14 +400,23 @@ namespace dxvk {
       dyInfo.pDynamicStates     = &dynamicState;
     }
 
+    VkPipelineCreateFlags flags = VK_PIPELINE_CREATE_LIBRARY_BIT_KHR;
+    if (m_device->features().extAttachmentFeedbackLoopLayout.attachmentFeedbackLoopLayout) {
```

Comment:
Do we need this check? The front-end should never set the bits if the feature is unsupported.

The logic here is that all pipeline state should be normalized against whatever the device supports *before* we end up looking up pipelines, see other things like conservative rasterization etc.